crates/bevy_pbr/src/render/light.rs

```diff
@@ -503,9 +503,9 @@ pub fn extract_lights(
 pub(crate) const POINT_LIGHT_NEAR_Z: f32 = 0.1f32;
 
 // Can't do `Vec3::Y * -1.0` because mul isn't const
-const NEGATIVE_X: Vec3 = const_vec3!([-1.0, 0.0, 0.0]);
-const NEGATIVE_Y: Vec3 = const_vec3!([0.0, -1.0, 0.0]);
-const NEGATIVE_Z: Vec3 = const_vec3!([0.0, 0.0, -1.0]);
+const NEGATIVE_X: Vec3 = Vec3::new(-1.0, 0.0, 0.0);
+const NEGATIVE_Y: Vec3 = Vec3::new(0.0, -1.0, 0.0);
+const NEGATIVE_Z: Vec3 = Vec3::new(0.0, 0.0, -1.0);
```

Comment:
`0.21` also added negative constants (`Vec3::NEG_X`, etc.)
We should use those.

Reply:
Should I remove these and replace all usages with `Vec3::*` equivalents? Or just redefine them? I've pushed redefinition.